src/pymor/algorithms/lyapunov.py

```diff
@@ -9,16 +9,21 @@
 from pymor.core.defaults import defaults
 from pymor.operators.interface import Operator
 
-_DEFAULT_LYAP_LRCF_SPARSE_SOLVER_BACKEND = ('pymess' if config.HAVE_PYMESS else
-                                            'lradi')
+_DEFAULT_CONT_LYAP_LRCF_SPARSE_SOLVER_BACKEND = ('pymess' if config.HAVE_PYMESS else 'lradi')
 
-_DEFAULT_LYAP_LRCF_DENSE_SOLVER_BACKEND = ('pymess' if config.HAVE_PYMESS else
+_DEFAULT_CONT_LYAP_LRCF_DENSE_SOLVER_BACKEND = ('pymess' if config.HAVE_PYMESS else
+                                                'slycot' if config.HAVE_SLYCOT else
+                                                'scipy')
+
+_DEFAULT_CONT_LYAP_DENSE_SOLVER_BACKEND = ('pymess' if config.HAVE_PYMESS else
                                            'slycot' if config.HAVE_SLYCOT else
                                            'scipy')
 
-_DEFAULT_LYAP_DENSE_SOLVER_BACKEND = ('pymess' if config.HAVE_PYMESS else
-                                      'slycot' if config.HAVE_SLYCOT else
-                                      'scipy')
+_DEFAULT_DISC_LYAP_LRCF_SPARSE_SOLVER_BACKEND = NotImplemented
+
+_DEFAULT_DISC_LYAP_LRCF_DENSE_SOLVER_BACKEND = ('slycot' if config.HAVE_SLYCOT else 'scipy')
+
+_DEFAULT_DISC_LYAP_DENSE_SOLVER_BACKEND = ('slycot' if config.HAVE_SLYCOT else 'scipy')
```

Comment:
is the number of defaults with very long names getting out of hand?

Reply:
I have actually omitted the 'lrcf' option, because I could not think of a name for "the opposite of lrcf" and that option was identical to non-lrcf dense/sparse anyway.